### DISCUSSION TITLE
Problems with classes in JIT

### DISCUSSION THREAD

**Discussion by mlamota2020** [tailwindlabs/tailwindcss#5480] [Help]
### What version of Tailwind CSS are you using?

v2.2.14 (exactly the version with PostCSS 7 compat)

### What build tool (or framework if it abstracts the build tool) are you using?

React v17.0.2

### What version of Node.js are you using?

v16.5.0

### What browser are you using?

Chrome and Firefox Developer Edition (but most Chrome)

### What operating system are you using?

Windows 10 v21H1

### Reproduction repository

https://gist.github.com/mlamota2020/eaacdd795c5279f4cdd4912fea371121

### Describe your issue

Well, the problem is that I'm using JIT mode and Headless UI for a project for school on React, and the problem is that the switch is having CSS and JS problems. I'm using it as a dark mode toggler.
https://www.loom.com/share/56f87c695c904742afebec23aaa430b4

**Comment by adamwathan** [comment]
Hey! You'll have to provide a real reproduction (a GitHub repo actually running React) — the classes you have all are correct and are picked up as expected in this static demo:

https://play.tailwindcss.com/dMBDNbRUAv

Going to convert to a discussion since I don't believe this is an actual bug but still happy to help you figure it out.
